canonical-web-and-design/ubuntu.com
# Portuguese Vmware to OpenStack takeover and engage page

## Done

- New Portuguese Vmware to OpenStack takeover and engage page

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/ and http://0.0.0.0:8001/engage/pt/vmware-para-o-openstack
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the takeover and engage page copy is correct against the [brief](https://docs.google.com/spreadsheets/d/1R1RV2n41mAzAHyQLKV6AQFmFfNFdnzQgwpz5CZY7JTI/edit#gid=2113339190) and [copy sheet](https://docs.google.com/spreadsheets/d/1p0RwL4uZqSh1AA1-bxgz9TyjHLm4eic-SNrzdMCtEl8/edit#gid=1373451093)

## Issue / Card

Fixes #7943

## Screenshots

takeover
![image](https://user-images.githubusercontent.com/441217/88148207-2caab600-cbf6-11ea-8625-09d9a40d0566.png)

engage page
![image](https://user-images.githubusercontent.com/441217/88148306-519f2900-cbf6-11ea-9196-6dcc65271d62.png)
